go-controller/pkg/util/ovs.go

```diff
@@ -895,3 +895,14 @@ func GetOVSPortPodInfo(hostIfName string) (bool, string, string, error) {
 	}
 	return true, sandbox, nadName, nil
 }
+
+// GetLogicalRouterSNATConntrackZone gets the ct-zone-[router]_snat field from
+// logical router external_ids, usually needed to discover what ct zones
+// the ovn-controller choose
+func GetLogicalRouterSNATConntrackZone(logicalRouterName string) (string, error) {
```

Comment:
is this used somewhere today or added for debugging purposes alone? even then how do we plan to leverage this? please add some details in commit message.

Reply:
Expanded commit message with 
```
 This functions will be integrated in the future with ovnkube-trace
    or/and klog to print ct zones selected by OVN.
```

Reply:
Dropping this comit.